exercises/isogram/isogram_test.py

```diff
@@ -25,6 +25,9 @@ def test_longest_reported_english_isogram(self):
     def test_word_with_duplicated_character_in_mixed_case(self):
         self.assertIs(is_isogram("Alphabet"), False)
 
+    def test_word_with_duplicated_character_in_mixed_case_lowercase_first(self):
```

Comment:
This line is too long. In cases where the canonical test name is too long to use in its exact form, it is acceptable to reword the test name. Might I suggest `test_word_with_duplicated_letter_in_mixed_case_lowercase_first`?